packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -290,6 +315,76 @@ export class JobService {
     return true;
   }
 
+  public async cancelJob(jobEntity: JobEntity): Promise<boolean> {
```

Comment:
this method does not exist any more in develop, it shouldn't be here, please check it